Directory.Build.props

```diff
@@ -47,4 +47,16 @@
 
   <!-- Build with the hybrid CRT (Universal CRT + Static VS CRT (for what little the Universal CRT doesn't cover) -->
   <Import Project="$(MSBuildThisFileDirectory)HybridCRT.props" />
+
```

Comment:
Should we have a <Error Text="Missing cert stuff, please run $(RepoRoot)\tools\DevCheck.cmd!" Condition="!Exists('$(RepoRoot)\.user')" /> or something like that here?